packages/cli/test/functional/test_site_templates/test_default/expected/stylesheets/main.css

```diff
@@ -118,6 +118,10 @@ mark {
 
 /* Bootstrap small(sm) responsive breakpoint */
 @media (max-width: 767.98px) {
+    .indented {
```

Comment:
let's replicate this in all the other ones containing the `.indented` class too for consistency (especially the docs)

Reply:
Thanks for pointing that out!
I think I forgot to modify them as well when I was updating the test files 😅